packages/core/getBusyTimes.ts

```diff
@@ -85,10 +87,15 @@ export async function getBusyTimes(params: {
   const endTimeDate =
     rescheduleUid && duration ? dayjs(endTime).add(duration, "minute").toDate() : new Date(endTime);
 
+  // to also get bookings that are outside of start and end time, but the buffer falls within the start and end time
+  const maxBuffer = getDefinedBufferTimes()[getDefinedBufferTimes().length - 1];
```

Comment:
NIT - saves calling this twice (Very small nit its lit a function returning a very small array its not expensive)
```suggestion
const definedBufferTimes = getDefinedBufferTimes()
  const maxBuffer = definedBufferTimes[definedBufferTimes.length - 1];
```